.github/workflows/examples.yaml

```diff
@@ -71,7 +71,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        folder: ['packages/angular_bloc/example']
+        folder: ['packages/bloc/example', 'package/bloc_test/example', 'examples/angular_counter']
```

Comment:
Dont forget to include and exclude `packages/*/example` to paths in both CI files :-)

Reply:
like this:

```
on:
  push:
    paths:
    - 'sub-project/**'
    - '!sub-project/docs/**'
```

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#example-using-positive-and-negative-patterns-1

Reply:
Good catch, thanks for the suggestion and snippet @tenhobi!
For `examples.yaml`, I was thinking we may want to run the CI when anything in packages/** gets modified so that we rerun the examples on library changes to make sure the examples are updated, what do you think?

Reply:
As the examples use specific version, nothing should really change. So I would opt out the path.

Reply:
Sounds good, I've whitelisted the package examples from the examples workflow.